esphome/components/remote_transmitter/automation.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#include "esphome/core/component.h"
+#include "esphome/core/automation.h"
+#include "esphome/components/remote_transmitter/remote_transmitter.h"
```

Comment:
Alphabetical, and also adds helpers for `Parented`
```suggestion
#include "esphome/components/remote_transmitter/remote_transmitter.h"
#include "esphome/core/automation.h"
#include "esphome/core/component.h"
#include "esphome/core/helpers.h"
```